### PR TITLE
Issue #200 Integrate with PCP

### LINKED ISSUE #200: Integrate with PCP
See [Performance Co-Pilot](http://pcp.io)

* https://github.com/fabric8-services/fabric8-notification/blob/master/pcp.repo
* https://github.com/fabric8-services/fabric8-notification/blob/master/notification%2Bpmcd.sh 
* https://github.com/fabric8-services/fabric8-notification/blob/master/Dockerfile.deploy#L13-L19 
* https://github.com/fabric8-services/fabric8-notification/blob/master/Dockerfile.deploy#L26

### PR DESCRIPTION
**Pull request by chmouel**
Importing what I have been doing on idler side,

Fixes #200

### CODE REVIEW

**Comment by fche** [comment]
fwiw lgtm

**Comment by aslakknutsen** [comment]
[test]

File: internal/router/router.go [hunk 1]
```diff
@@ -3,12 +3,14 @@ package router
...
 	"github.com/julienschmidt/httprouter"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 // CreateAPIRouter is creating a router for the REST API of the Proxy.
 func CreateAPIRouter(api api.ProxyAPI) *httprouter.Router {
 	// Create router for API
 	proxyRouter := httprouter.New()
 	proxyRouter.GET("/api/info/:namespace", api.Info)
+	proxyRouter.Handler("GET", "/metrics", prometheus.Handler())
```
[the remark this answers is not in the document]
Comment (chmouel) [reply internal/router/router.go]: Every other services are `/metrics`, why would you want to change it for the sake of changing instead of uniformity ?

**Comment by chmouel** [comment]
@kishansagathiya answered on your first concern on Mattermost

File: internal/router/router.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (kishansagathiya) [reply internal/router/router.go]: cool, makes sense.

**Comment by kishansagathiya** [comment]
@chmouel I just tried this again. Same results.
This is what I have been doing. 
```
> docker image build -t pcp -f Dockerfile.deploy .
> export DSAAS_PREVIEW_TOKEN=<dsaas-preview token>
> ./scripts/setupLocalProxy.sh start
> eval $(./scripts/setupLocalProxy.sh env)
> fabric8-jenkins-proxy
> docker run pcp
```
Is this the way?
Is it possible to do this as a part of the shell script that we have?

**Comment by chmouel** [comment]
@kishansagathiya how to test jenkins-proxy has nothing to do with this review, can you log a new issue? (and as you know i have done mostly jenkins idler work, since you are taking care of most of the jenkins-proxy issues i would expect you know that answer, because i don't)

**Comment by chmouel** [comment]
it's tested in the unittests fyi ;)

**Pull request merged**